README.md

```diff
@@ -1,8 +1,87 @@
 # Interoperable Immunization Data Initiative (IIDI)
 
-## One-liner summary
+## One-liner Summary
 
-TODO
+A structured implementation for **interoperable immunization data exchange**, supporting **FHIR-based APIs**, federated aggregation, and simulated cross-jurisdictional record transfers.
+
+---
+
+## Purpose
+
+The **Interoperable Immunization Data Initiative (IIDI)** provides a **development environment** for testing **data interoperability, aggregation, and transfer mechanisms** between simulated jurisdictions. The initiative focuses on **FHIR-compliant data exchange**, ensuring that immunization records remain **structured, standardized, and accessible** while respecting system independence.
+
+---
+
+## Current Development Scope (MVP)
+
+This repository implements the **minimum viable product (MVP)**, which includes:
+
+- **FHIR-compliant immunization data repositories**, preloaded with synthetic patient records.
+- **Mock provincial services**, simulating jurisdictional immunization data endpoints.
+- **Federated data aggregation services** to generate structured immunization reports.
+- **Basic PT-to-PT record transfer simulation** using FHIR standards.
+
+---
+
+## Technical Architecture
+
+The IIDI architecture follows a **modular and federated approach**, ensuring **flexibility, security, and interoperability**:
+
+## Key Architectural Components
```

Comment:
Could add "Integrity & Fault-Tolerance"? That might be close to over promising, but in theory it's fundamental to a data system and if we can do it we should (I'd say "maybe" given time and resources)